docs/editor/accessibility.md

```diff
@@ -9,59 +9,59 @@ MetaDescription: Visual Studio Code user accessibility features.  Learn here abo
 ---
 # Accessibility
 
-Visual Studio Code has many features to help make the editor accessible to all users. Zoom and High Contrast colors improve editor visibility, keyboard-only navigation allows use without a mouse, and the editor has been optimized for screen readers.
+Visual Studio Code has many features to help make the editor accessible to all users. Zoom levels and High Contrast colors improve editor visibility, keyboard-only navigation supports use without a mouse, and the editor has been optimized for screen readers.
 
 ## Zoom
 
-You can adjust the Zoom level in VS Code with the **View** > **Appearance** > **Zoom** commands.  The zoom level increases or decreases by 20% each time a **Zoom** command is executed.
+You can adjust the zoom level in VS Code with the **View** > **Appearance** > **Zoom** commands.  Each **Zoom** command will increase or decrease the zoom level by 20 percent.
 
-* **View** > **Appearance** > **Zoom In** (`kb(workbench.action.zoomIn)`) - increase the Zoom level.
-* **View** > **Appearance** > **Zoom Out** (`kb(workbench.action.zoomOut)`) - decrease the Zoom level.
-* **View** > **Appearance** > **Reset Zoom** (`kb(workbench.action.zoomReset)`) - reset the Zoom level to 0.
+* **View** > **Appearance** > **Zoom In** (`kb(workbench.action.zoomIn)`) - increase the zoom level.
+* **View** > **Appearance** > **Zoom Out** (`kb(workbench.action.zoomOut)`) - decrease the zoom level.
+* **View** > **Appearance** > **Reset Zoom** (`kb(workbench.action.zoomReset)`) - reset the zoom level to 0.
 
->**Note**: If you are using a magnifier make sure to hold the `alt` key while viewing the hover to allow the mouse to move over the hover.
+>**Note**: If you're using a magnifier, hold down the `alt` key while viewing the hover to move the cursor over the hover.
 
 ![Zoomed in editor](images/accessibility/zoomed-in.png)
 
-### Persisted Zoom Level
+### Persistent zoom level
 
-When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
+When you adjust the zoom level with the **View** > **Zoom In** or  **Zoom Out** commands, the zoom level is persisted in the [`window.zoomLevel` setting](/docs/getstarted/settings.md). The default value is 0, and each increment or decrement changes the zoom level by 20 percent.
```

Comment:
code blocks in links don't render well on the site and the link is general to settings, not specific to window.zoomLevel.
I'd revert this change

Reply:
I all the topics, we deliberately add a link to docs/getstarted/settings.md on the first mention of a user setting.
We found that this increased the page views for that important Get Started page.